### DISCUSSION TITLE
select组件在选中时，鼠标移到其他项，原选中项的状态消失了

### DISCUSSION THREAD

**Discussion by asher886** [arco-design/arco-design-vue#1799] [Q&A] [answered]
select组件在选中时，鼠标移到其他项，原选中项的状态消失了

**Comment by cgrain1998** [comment]
+1
想实现下面这种效果
 https://codepen.io/pen?&editors=001

**Comment by hehehai** [comment]
> +1 想实现下面这种效果 [codepen.io/pen?&editors=001](https://codepen.io/pen?&editors=001)

@cgrain1998 hi, 这个链接好像是空的。

**Comment by cgrain1998** [comment]
> > +1 想实现下面这种效果 [codepen.io/pen?&editors=001](https://codepen.io/pen?&editors=001)
> 
> @cgrain1998 hi, 这个链接好像是空的。

大佬，其实他就是 ant design  框架的 下拉框
![image](https://user-images.githubusercontent.com/35121432/195484033-146ad453-b112-4c81-8ccf-169de13d8754.png)


您可以参考下看看。
https://antdv.com/components/select


鼠标移动上去 activate 是不会是去焦点样式的，现在arco 的框架 它虽然也有这个效果，但是鼠标一放上去，他就没了~~~ 您看看这东西是不是bug ，或者有没有觉得这个有点不太合理  （arco design 是一个nb的框架 ！！！）

**Comment by hehehai** [comment, marked as answer]
> select组件在选中时，鼠标移到其他项，原选中项的状态消失了

@asher886 hi, 目前 select 的组建选择交互是这样设计的。当前选中项是会出现在选择框内的，预选列表在发生其他预选项交互后就不做选中高亮了。参考 [demo](https://codesandbox.io/s/heuristic-mountain-ff1cgc?file=/src/App.vue)

**Comment by hehehai** [comment]
@cgrain1998 嗯，明白的。确实这样会方便些，不过目前 Arco 的 UX 设计不好调整。这块想着可以内容加一个 预选列表选中项的 className，这样可以全局的做主题样式处理，不需要每个组件都单独处理了。

@flsion 你觉得呢？有什么思路吗？

**Comment by cgrain1998** [comment]
> @cgrain1998 嗯，明白的。确实这样会方便些，不过目前 Arco 的 UX 设计不好调整。这块想着可以内容加一个 预选列表选中项的 className，这样可以全局的做主题样式处理，不需要每个组件都单独处理了。
> 
> @flsion 你觉得呢？有什么思路吗？


好的，静候佳音

**Comment by flsion** [comment]
> @cgrain1998 嗯，明白的。确实这样会方便些，不过目前 Arco 的 UX 设计不好调整。这块想着可以内容加一个 预选列表选中项的 className，这样可以全局的做主题样式处理，不需要每个组件都单独处理了。
> 
> @flsion 你觉得呢？有什么思路吗？

嗯，目前 UX 的设计是这样的，参考两种效果我感觉区分下 已选择 和 hover效果 会更好一些。我来和设计反馈一下，这个调整涉及的组件和内容可能会多一些。

**Comment by cgrain1998** [comment]
> > @cgrain1998 嗯，明白的。确实这样会方便些，不过目前 Arco 的 UX 设计不好调整。这块想着可以内容加一个 预选列表选中项的 className，这样可以全局的做主题样式处理，不需要每个组件都单独处理了。
> > @flsion 你觉得呢？有什么思路吗？
> 
> 嗯，目前 UX 的设计是这样的，参考两种效果我感觉区分下 已选择 和 hover效果 会更好一些。我来和设计反馈一下，这个调整涉及的组件和内容可能会多一些。


锤他   +1  / 手动滑稽

**Comment by OverTree** [comment]
静候佳音 +1
